db/sqldb/migrations/0013_user_search.up.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
+
+CREATE EXTENSION IF NOT EXISTS pg_trgm;
+SET pg_trgm.similarity_threshold = 0.1;
```

Comment:
What does setting this at migration-time do? 

Reading [the docs](https://www.postgresql.org/docs/current/pgtrgm.html), I think you'd actually want to either set it on our connection pool, or on a per query basis (either in a transaction or with `SET LOCAL`)

Reply:
Sorry I actually no longer need this - I was initially doing a fuzzy search, but I pivoted to doing an `ILIKE` search, so we no longer need similarity threshold.